airflow/providers/google/cloud/log/gcs_task_handler.py

```diff
@@ -153,6 +153,10 @@ def _read(self, ti, try_number, metadata=None):
         try:
             blob = storage.Blob.from_string(remote_loc, self.client)
             remote_log = blob.download_as_string()
+
+            if isinstance(remote_log, bytes):
+                remote_log = remote_log.decode()
```

Comment:
I would say that this should be part of `download_as_string` method. The name suggest that the output should be a `str` not `bytes` so fixing it there may solve some issues in future.

Reply:
You're right, looks like the method is deprecated and it is now an alias for `download_as_byte`

https://googleapis.dev/python/storage/latest/blobs.html#google.cloud.storage.blob.Blob.download_as_string

Will update the PR